README.md

```diff
@@ -36,7 +36,7 @@ Our code of conduct is [Contributor Covenant](CODE_OF_CONDUCT.md). All contribut
 
 ## Maintainers
 
-Reto Kromer (@retokromer)
+vacat
```

Comment:
Shouldn't it be "vacant"?

Reply:
Why? Thanks G’d I am only one person.

Reply:
I was wondering if it was just a spelling mistake: "vacat" instead of "vacant".

Reply:
Vacat is singular, vacant is plural. I replaced one name. 
(That said, the title was always wrong, because I never could find a co-maintainer ;-)

Reply:
Interesting. I didn't know that. Is it common in English? Because adjectives in English have no plural or singular.
Anyway, I thought it was a typo and couldn't find any references in any dictionary.

Reply:
I’m not an English speaker. This is a Latin expression. In Switzerland it’s a common way to indicate an empty position, e.g. in academia.

Reply:
Thanks for the info. I guessed it was Latin, but it was the first time I saw it used that way in this conjugated form (even if I'm Swiss as well).
I guess we never stop learning. :-)

Too bad there is no maintainer taking over. Thanks for the work!